File: app/src/main/java/com/cappielloantonio/play/subsonic/models/Genre.java
package com.cappielloantonio.play.subsonic.models;

import com.tickaroo.tikxml.annotation.Attribute;
import com.tickaroo.tikxml.annotation.TextContent;
import com.tickaroo.tikxml.annotation.Xml;

@Xml
public class Genre {
    @TextContent
    protected String genre;
    @Attribute
    protected int songCount;
    @Attribute
    protected int albumCount;

    public String getGenre() {
        return genre;
    }

    public void setGenre(String value) {
        this.genre = value;
    }

    public int getSongCount() {
        return songCount;
    }

    public void setSongCount(int value) {
        this.songCount = value;
    }

    public int getAlbumCount() {
        return albumCount;
    }

    public void setAlbumCount(int value) {
        this.albumCount = value;
    }
}

File: app/src/main/java/com/cappielloantonio/play/subsonic/models/Genre.kt
package com.cappielloantonio.play.subsonic.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Genre : Parcelable {
    @SerializedName("value")
    var genre: String? = null
    var songCount = 0
    var albumCount = 0
}